site/src/components/pages/user/utils.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Insert shy hypen indicators at capital letters within single words to improve wrapping ability
+ */
+export const shy = (text: string) =>
```

Comment:
I’d call this function `insertShyHyphensAtCaptitalLeters`, but that’s too minor. 🙂 Maybe we can do this later if we need to hoist reuse the function elsewhere.